File: support_tools/swigwin-3.0.12/Examples/test-suite/csharp/constover_runme.cs
using System;
using constoverNamespace;

public class runme
{
    static void Main() 
    {
        string p = constover.test("test");
        if ( p != "test" )
            throw new Exception( "test failed!" );

        p = constover.test_pconst("test");
        if ( p != "test_pconst" )
            throw new Exception( "test_pconst failed!" );

        Foo f = new Foo();
        p = f.test("test");
        if ( p != "test" )
            throw new Exception( "member-test failed!" );

        p = f.test_pconst("test");
        if ( p != "test_pconst" )
            throw new Exception( "member-test_pconst failed!" );

        p = f.test_constm("test");
        if ( p != "test_constmethod" )
            throw new Exception( "member-test_constm failed!" );

        p = f.test_pconstm("test");
        if ( p != "test_pconstmethod" )
            throw new Exception( "member-test_pconstm failed!" );
    }
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/constover_runme.m
# do not dump Octave core
if exist("crash_dumps_octave_core", "builtin")
  crash_dumps_octave_core(0);
endif

constover

p = constover.test("test");
if (!strcmp(p,"test"))
    error("test failed!")
endif

p = constover.test_pconst("test");
if (!strcmp(p,"test_pconst"))
    error("test_pconst failed!")
endif
    
f = constover.Foo();
p = f.test("test");
if (!strcmp(p,"test"))
    error("member-test failed!")
endif

p = f.test_pconst("test");
if (!strcmp(p,"test_pconst"))
    error("member-test_pconst failed!")
endif

p = f.test_constm("test");
if (!strcmp(p,"test_constmethod"))
    error("member-test_constm failed!")
endif

p = f.test_pconstm("test");
if (!strcmp(p,"test_pconstmethod"))
    error("member-test_pconstm failed!")
endif


